rmake.py

```diff
@@ -210,8 +216,20 @@ def config_cmd():
     create_dir( os.path.join(build_path, "clients") )
     os.chdir( build_path )
 
+    # compiler
+    if args.use_hipcc_compiler:
+        cmake_options.append(f"-DCMAKE_C_COMPILER={rocm_path}/bin/hipcc")
+        cmake_options.append(f"-DCMAKE_CXX_COMPILER={rocm_path}/bin/hipcc")
+    else:
+        cmake_options.append(f"-DCMAKE_C_COMPILER=gcc")
+        cmake_options.append(f"-DCMAKE_CXX_COMPILER=g++")
```

Comment:
Sorry I can't recall how this builds on windows but would some or all of this be in a not os "nt" case?

Reply:
I see it's set to clang++ in [toolchain-windows.cmake](https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/toolchain-windows.cmake#L13), so it takes clang++ regardless, is that right?
Either way, I'll fiddle around on windows and update accordingly.

Reply:
Well if you know it get's overridden it may confuse to explicitly set on command line, did windows CI run ?

Reply:
CI ran and passed. I updated it a bit to note that hipblas currently can't be build without hip-clang. Can try building hipBLAS with other compilers on windows at some point if we decide it's worth pursuing.